### PR TITLE
Rewrite Tykky page from user perspective

### PR DESCRIPTION
**Pull request by ktiits**

### CODE REVIEW

**Comment by ktiits** [comment]
Added quite a lot of text that could be useful for end-users. Hopefully all correct.
https://csc-guide-preview.rahtiapp.fi/origin/ktiits-tykky/computing/containers/tykky/

File: docs/computing/containers/tykky.md [hunk 1]
```diff
@@ -20,79 +27,80 @@ are not covered here yet.
...
+before using the command. 
 
-or 
-```
-conda list -n <target_env_name> --explicit > env.txt
+- [Miniconda end user license agreement](https://www.anaconda.com/end-user-license-agreement-miniconda)
+- [Anaconda terms of service](https://www.anaconda.com/terms-of-service)
+- [A blog entry on Anaconda commercial edition](https://www.anaconda.com/blog/anaconda-commercial-edition-faq)
+
+1) Create **conda environment file** env.yml:
```
Comment (JMuff22) [inline docs/computing/containers/tykky.md:55]: An example `env.yaml` may be useful for the user here.

Comment (JMuff22) [reply docs/computing/containers/tykky.md:55]: Also `conda env export -n <target_env_name> > env.yaml` quick instructions to export an environment.

File: docs/computing/containers/tykky.md [hunk 2]
```diff
@@ -20,79 +27,80 @@ are not covered here yet.
...
 
 ```
 conda-containerize new -r req.txt --prefix <install_dir> env.yml
 ```
 
+### mamba 
 The tool also supports using [mamba](https://github.com/mamba-org/mamba) 
-for installing packages. Enable this feature by adding the `--mamba` flag. 
-`conda-containerize new --mamba ...`
+for installing packages. Mamba often findes suitable packages much faster than conda, so it is a good option when required package list is long. Enable this feature by adding the `--mamba` flag.
```
Comment (JMuff22) [inline docs/computing/containers/tykky.md:88]: findes > finds

File: docs/computing/containers/tykky.md [hunk 3]
```diff
@@ -20,79 +27,80 @@ are not covered here yet.
...
 
-- [Miniconda end user license agreement](https://www.anaconda.com/end-user-license-agreement-miniconda)
-- [Anaconda terms of service](https://www.anaconda.com/terms-of-service)
-- [A blog entry on Anaconda commercial edition](https://www.anaconda.com/blog/anaconda-commercial-edition-faq)
 
 ### End-to-end example 
 
-Using the previous `env.yml`
-
+Create new conda based installation.
```
Comment (JMuff22) [inline docs/computing/containers/tykky.md:97]: "...using the previous `env.yml` example"

File: docs/computing/containers/tykky.md [hunk 4]
```diff
@@ -107,12 +115,10 @@ Type "help", "copyright", "credits" or "license" for more information.
...
-
-## Modifying a conda installation
-
-As Tykky installed software resides in a container, it can not be directly modified.
-Small python packages can be added normally using `pip`, but then the python packages are
+Tykky installed software resides in a container, so it can not be directly modified.
+Small Python packages can be added normally using `pip`, but then the Python packages are
 sitting on the parallel filesystem so this is not recommended for any larger installations.  
 
 To actually modify the installation we can use the `update` keyword
```
Comment (JMuff22) [inline docs/computing/containers/tykky.md:124]:
```
conda  install -y numpy
conda  remove -y pyyaml
pip install requests
```
With the new RHEL8 update, will conda installations in the  `--post-install` bash script work?

Comment (ktiits) [reply docs/computing/containers/tykky.md:124]: @Nortamo ??

**Comment by ktiits** [comment]
I will be on holidays now for 1 week. Did not add yml file example yet, but of course it could be added.

File: docs/computing/containers/tykky.md (hunk 4 above)
Comment (JMuff22) [reply docs/computing/containers/tykky.md:124]: Ok end-to-end example still works :)

Comment (JMuff22) [reply docs/computing/containers/tykky.md:124]: Errors on Mahti but not Puhti `MyEnv/bin/tput: Argument list too long` . Line 44,46,47,48,49 in `/bin/../common.sh`. But `--post-install` still works fine.

**Review by JMuff22 — APPROVED** [review]

**Pull request merged**